src/lib/core/TLVWriter.h

```diff
@@ -128,14 +128,11 @@ class DLL_EXPORT TLVWriter
      *
      * @retval #CHIP_NO_ERROR        Successfully reserved required buffer size.
      * @retval #CHIP_ERROR_NO_MEMORY The reserved buffer size cannot fits into the remaining buffer size.
+     * @retval #CHIP_ERROR_INCORRECT_STATE
+     *                               Uses TLVBackingStore and is in a state where it might allocate
+     *                               additional non-contigious memory, thus making it impossible to
```

Comment:
may need some rephrasing `making it impossible to` feels like incomplete

Reply:
Ops, forgot to finish my thought. Fixed now